src/hotspot/share/gc/shenandoah/shenandoahHeap.inline.hpp

```diff
@@ -658,11 +658,11 @@ inline bool ShenandoahHeap::has_evacuation_reserve_quantities() const {
 }
 
 inline bool ShenandoahHeap::is_idle() const {
-  return _gc_state.is_unset(YOUNG_MARKING | OLD_MARKING | EVACUATION | UPDATEREFS);
+  return _gc_state.is_unset(MARKING | YOUNG_MARKING | OLD_MARKING | EVACUATION | UPDATEREFS);
```

Comment:
We should just need to check `MARKING` now, right?

Reply:
true.  (I guess I can also add assert (!MARKING || (YOUNG_MARKING || OLD_MARKING)).  I'll make this change.